index.html

```diff
@@ -0,0 +1,321 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Mobile Website</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <div class="toolbar">
+      <nav class="nav-bar">
+        <a href="#"><img src="./images/logo.png" alt="logo" /></a>
+        <ul class="sid-cons">
+          <li>Portoflio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+      </nav>
+    <div class="headline-container">
+      <div class="items1">
```

Comment:
- [ ] Please fix horizontal scrolling in mobile view as it is not good for UX 🙏 
- [ ] On mobile view, show a hamburger button instead of the menu in a column like you have done here.
### Your Design
![image](https://user-images.githubusercontent.com/46839548/166106247-5cbe4338-fcec-4e6c-9d74-0558fe286b77.png)
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/166106269-2552bae8-bf28-4207-835a-eb5101df4c64.png)